### PR TITLE
Monorepo. 7 modules with 1 common module.

### PR DESCRIPTION
**Pull request by shamil-gadelshin**
Converted modules from inner folders to inner crates:
- common (contains widely used GovernanceCurrency)
- membership
- roles
- storage
- service-discovery
- memo
- content-working-group
- governance

Comments:
- `content-working-group` module wasn't tested because of the forum `module` dependency issue
- the `runtime` itself builds successfully, but the `node` cannot be built because of the same issue with `forum` module

### CODE REVIEW

File: runtime-modules/content-working-group/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+package = 'substrate-primitives'
+rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
+
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '1.0.0'
+
+[dependencies.forum]
```
Comment (mnaamani) [inline runtime-modules/content-working-group/Cargo.toml]: I think dependency on forum is only for use of the validation method, `InputValidationLengthConstraint` that should probably be moved to the common crate later.

**Comment by shamil-gadelshin** [comment]
`cargo fmt` and `cargo test` commands work recursively when applied on the workspace Cargo.toml.

I tested it on my proposals module workspace with three modules.

File: runtime-modules/content-working-group/Cargo.toml (hunk 1 above)
Comment (shamil-gadelshin) [reply runtime-modules/content-working-group/Cargo.toml]: https://github.com/Joystream/substrate-runtime-joystream/issues/184

**Comment by mnaamani** [comment]
So we need to turn our root Cargo.toml setup to a `[workspace]` because at the moment it is just a library package, which I think comes next in your plan.

**Comment by mnaamani** [comment]
> Confirmed Runtime and Node builds successfully.
> Lets merge after #186 and #185

Oh I forgot this is towards monorepo branch. Will merge.

**Pull request merged**